data/hai/unfettered 0 prologue.txt

```diff
@@ -297,6 +297,127 @@ mission "Unfettered Jump Drive 3"
 
 
 
+mission "Unfettered Jump Drive Trading"
+	minor
+	landing
+	invisible
+	source "Darkcloak"
+	to offer
+		has "Unfettered Jump Drive 3: offered"
+		not "Wanderers: Jump Drive Source: active"
+		# There is no minimum amount of JD to sell (other than the first 3), except if you failed to prove yourself the first time.
+		# "Unfettered Jump Drive 4: failed" represents the amount of JD sold.
+		"unfettered: jd to sell" < "Unfettered Jump Drive 4: failed"
+		or
+			# guaranteed if you sold more than 10 JD
+			"Unfettered Jump Drive 4: failed" > 10
+			random < 25
+			and
+				"Unfettered Jump Drive 4: failed" > 5
+				random < 50
+	on offer
+		conversation
+			`The Unfettered are starting to know you by reputation for bringing jump drives to them. This time, one of them - an engineer, judging by the writing pad and equipment they have - is waiting for you at your hatch. Maybe they mean to trade something other than money with you. Do you want to bargain for another drive with them?`
+			choice
+				`	(No.)`
+					defer
+				`	(Yes.)`
+			`	You exit your ship and immediately the engineer starts to lead you to the outfitter. "We will show you the weapon once you arrive. For now you will only be allowed to access one of them, but if you continue to prove of great assistance to us we may give you access to the others."`
```

Comment:
The introduction of trading for weapons is really abrupt here. The engineer just assumes that you're here for weapons and starts showing them to you immediately.

Reply:
Well the Unfettered cant imagine you wouldnt be interested in weapons

Reply:
I added an option where you can say you're not interested in hai weaps and they say you should see it first